internal/pkg/utils/testhelper/env.go

```diff
@@ -28,6 +30,22 @@ func ReplaceEnvsStringWithSeparator(str string, provider EnvProvider, envSeparat
 		})
 }
 
+// ReplaceEnvsStringWithSeparatorE replaces ENVs in given string with chosen separator.
+func ReplaceEnvsStringWithSeparatorE(str string, provider testtemplateinputs.TestInputsEnvProvider, envSeparator string) (string, error) {
```

Comment:
Zvykneme pouzivat (ak je viacero verzii)
`MustReplaceEnvsStringWithSeparator` -> panic
`ReplaceEnvsStringWithSeparator` -> err

Viem, ze to `E` tam ma CLI framework, ale nikde inde som to zatial nevidel, ... a to `Must` sa pouziva dost casto aj v inych libkach.

Reply:
Á, to je ono. Já si právě lámal hlavu s tím jak je odlišit a to `E` mi zůstalo v čerstvé paměti. 🙂 Upravím.

Reply:
https://github.com/keboola/keboola-as-code/pull/825/commits/abc9f5226cff59ad07ae428faf9fdd327e201a17 (znamená to teda, že jsem všechny existující kromě té nové oprefixoval `Must` 🙂 )

Reply:
super :rocket: